game/src/common/mod.rs

```diff
@@ -117,12 +117,11 @@ impl CommonState {
 
     fn osd_for(app: &App, id: ID) -> Text {
         let map = &app.primary.map;
-        let id_color = app.cs.bottom_bar_id;
         let mut osd = Text::new();
         match id {
             ID::Lane(l) => {
                 if app.opts.dev {
-                    osd.append(Line(l.to_string()).fg(id_color));
+                    osd.append(Line(l.to_string()).bold_body());
```

Comment:
Maybe we should swap the bold/underline for IDs/names; I don't have a strong preference. But the red was pretty awful. :P